packages/UnorderedList/UnorderedList.md

```diff
@@ -31,3 +31,18 @@
   <UnorderedList.Item>A mix of numbers, lowercase and uppercase letters</UnorderedList.Item>
 </UnorderedList>
 ```
+
+### Mixed List
+
+If you do not want each list item to have the same icon, you may give each item its own icon via the `itemStyle` prop. This will override a `listStyle` set by the parent.
```

Comment:
Note here, or in the general Lists.md file, usage criteria for mixed list styles:

- Mixed lists should only be used within Notification or InputFeedback components